content/zh-cn/docs/reference/using-api/_index.md

```diff
@@ -59,11 +59,11 @@ JSON 和 Protobuf 序列化模式遵循相同的模式更改原则。
 
 <!--
 The API versioning and software versioning are indirectly related.
-The [API and release versioning proposal](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md)
+The [API and release versioning proposal](https://git.k8s.io/design-proposals-archive/release/versioning.md)
 describes the relationship between API versioning and software versioning.
 -->
 API 版本控制和软件版本控制是间接相关的。
-[API 和发布版本控制提案](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md)
+[API 和发布版本控制提案](https://git.k8s.io/design-proposals-archive/release/versioning.md)
```

Comment:
Please see #34976:
> [release/versioning.md](https://git.k8s.io/design-proposals-archive/release/versioning.md#kubernetes-release-versioning) that was in the k/design-proposals-archive repository in the past has been migrated to [k/sig-release repository](https://github.com/kubernetes/sig-release/blob/master/release-engineering/versioning.md?rgh-link-date=2022-07-13T06%3A58%3A09Z) repository.

And the suggestion for this file will be applied by #34996.

Reply:
thanks!